src/pkg/podman/podman.go

```diff
@@ -29,6 +29,7 @@ import (
 
 var (
 	LogLevel = logrus.ErrorLevel
+	Version  = ""
```

Comment:
We shouldn't export this variable from the `podman` package. This pull request itself is a good example of why we shouldn't do this. :)

If clients of this package use the `GetVersion` function to get the version as a string, we have a lot more flexibility and elbow room to change the underlying implementation in future. eg., this pull request changes the implementation from always querying the version from Podman, to caching it.

Another example is https://github.com/containers/toolbox/pull/487 that changed the implementation of the `GetVersion` function in the `version` package without affecting any of the client code.

However, I see that we have a tricky naming problem - `version` is already taken by `github.com/HarryMichal/go-version`, and there's a `podmanVersion` that's local to the `CheckVersion` function. Maybe we can rename the variable in `CheckVersion` to `currentVersion` or something, it seems to match its `requiredVersion` counterpart, to free up a name?

Reply:
Nice catch! I did not realize the variable should not be exported.

Concerning the variable name, make whatever edits you think are right 👍.